### ISSUE TITLE
[a11y] Cannot create a new Journal

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1487]
I cannot create a new Journal. In `edge` I can create one both on the standalone form and from the QCBX.

![image](https://user-images.githubusercontent.com/37256050/167657032-8f0140c3-59c1-4446-b645-0fe302bc3132.png)

![msedge_Qn9W85SrDA](https://user-images.githubusercontent.com/37256050/167656754-516494e0-2534-4323-a337-3a7a24a61d8d.gif)

db: fwri
user: plarson
col: ichthyology
form: Reference Work/Journal

Windows 10, Edge

**Comment by maxxxxxdlp** [comment]
Duplicate of #1480
